formats/cbor/commonMain/src/kotlinx/io/Streams.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2017-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.io
```

Comment:
Maybe get rid of this package too? Place them in kotlinx.serialization.internal or kotlinx.serialization.cbor(.internal)